src/components/Tooltip/Tooltip.tsx

```diff
@@ -178,7 +178,8 @@ const TooltipAnchor: FC<TooltipAnchorProps> = ({
     if (!isValidElement(children)) return;
 
     if (isTriggerInteractive) {
-      const props = context.getReferenceProps({ ref });
+      // @ts-expect-error needed to fix https://github.com/element-hq/compound/issues/333
```

Comment:
What is the error here?

Reply:
`TS2698: Spread types may only be created from object types.`

Reply:
What is the type from `children.props`?
Is it specific to React <19?

Reply:
React 19: `children.props: unknown`
React 18: `children.props: any`

I can instead cast it to `children.props as object` to avoid the error

Reply:
I think doing some actual type testing might be saner, `...(typeof children.props === "object" ? children.props : {})`

Reply:
Props can be of any type, often you see a pattern of using function as children also.

Reply:
Oh, good idea